stable/mission-control/templates/mission-control-custom-secrets.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.common.customSecrets }}
+{{- range .Values.common.customSecrets }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .name }}
```

Comment:
@anastasiagrinman  can we please change this name  to `{{ template "mission-control.fullname" . }}-customsecrets`

Reply:
Done